packages/voting/src/voting.rs

```diff
@@ -59,6 +59,10 @@ impl MultipleChoiceVotes {
         self.vote_weights[vote.option_id as usize] += weight;
     }
 
+    pub fn remove_vote(&mut self, vote: MultipleChoiceVote, weight: Uint128) {
+        self.vote_weights[vote.option_id as usize] -= weight;
```

Comment:
we should probably use checked_sub here (and change it to checked_add in add_vote above)

Reply:
good one!